src/main/webapp/assets/js/utils.js

```diff
@@ -98,6 +98,17 @@ var utilPopWindow = function(url, name){
     return false;
 };
 
+/*used by modal (other instructor field) */
+var validateUniqname = function (value) {
+  var value = $.trim(value)
+  var letterOnly = /^[a-zA-Z]+$/;  
```

Comment:
Also, a valid uniqname is 3-8 alphabetic characters, so you might want to change the regexes (regices?) to:

/^[a-z]{3,8}$/i